src/main/kotlin/GameController.kt

```diff
@@ -11,16 +11,27 @@ class GameController : IController {
 
     override fun generateResponse(request: HttpRequest): HttpResponse {
         val httpResponse = HttpResponse()
-        val gameData = jsonParse.generateGameData(request.body.toString(Charset.defaultCharset()))
         try {
-            val board = tttEngineParser.generateBoard(gameData.board, gameData.input)
-            httpResponse.status = Status.OK
-            httpResponse.setBody(generateBody(board))
-        } catch (e: Exception) {
+            val gameData = jsonParse.generateGameData(request.body.toString(Charset.defaultCharset()))
+            try {
```

Comment:
What happens if the move the user sends to play is already taken?

Reply:
@td-extreme The API will return the same board it received.

Reply:
Does it send back an error message as well, telling the user the move was invalid??

Reply:
Hmmm, it doesn't send back an error message but I can definitely include that as part of the JSON. I still think it should said back the same board provided all the other input was correct

Reply:
yeah, I would want to know why I got the same board back.

Reply:
makes sense. I'll make that change